sycl/unittests/queue/GetProfilingInfo.cpp

```diff
@@ -429,18 +416,54 @@ TEST(GetProfilingInfo, partial_profiling_workaround) {
     cgh.parallel_for<InfoTestKernel>(globalWIs, [=](sycl::id<1> idx) {});
   });
   event.wait();
-  try {
-    event.get_profiling_info<sycl::info::event_profiling::command_submit>();
-    FAIL() << "No exception was thrown";
-  } catch (sycl::exception &e) {
-    EXPECT_EQ(e.code(), sycl::errc::invalid);
-    EXPECT_STREQ(
-        e.what(),
-        "Submit profiling information is temporarily unsupported on this "
-        "device. This is indicated by the lack of queue_profiling aspect, but, "
-        "as a temporary workaround, profiling can still be enabled to use "
-        "command_start and command_end profiling info.");
-  }
-  event.get_profiling_info<sycl::info::event_profiling::command_start>();
-  event.get_profiling_info<sycl::info::event_profiling::command_end>();
+  auto submit_time =
+      event.get_profiling_info<sycl::info::event_profiling::command_submit>();
+  auto start_time =
+      event.get_profiling_info<sycl::info::event_profiling::command_start>();
+  auto end_time =
+      event.get_profiling_info<sycl::info::event_profiling::command_end>();
+  (void)submit_time;
+  (void)start_time;
+  (void)end_time;
+}
```

Comment:
What does this test checks? Can we check the validness of submit_time, start_time and end_time?

Reply:
Yes, we can do that. Let me add